README.md

```diff
@@ -33,12 +33,18 @@ helm install --name=kafka-minion kafka-minion/kafka-minion
 | `kafka.brokers` | Comma delimited list of brokers to connect to | (none) |
 | `kafka.sasl.enabled` | Bool to enable/disable SASL authentication (only SASL_PLAINTEXT is supported) | `false` |
 | `kafka.sasl.useHandshake` | Whether or not to send the Kafka SASL handshake first | `true` |
-| `kafka.sasl.credentials.existingSecret` | Secretname of an existing secret which contains SASL credentials | (none) |
+| `kafka.sasl.existingSecret` | Secretname of an existing secret which contains SASL credentials | (none) |
```

Comment:
Unfortunately this is not correct. Apparently I was inconsistent with the naming between the SASL and TLS credentials. I can not merge this because this would be a breaking change :(. I'm sorry